drivers/volume/portworx/portworx.go

```diff
@@ -1453,6 +1454,77 @@ func (d *portworx) ValidatePureFaFbMountOptions(volumeName string, mountoption [
 
 }
 
+func (d *portworx) ValidatePureFaCreateOptions(volumeName string, FStype string, volumeNode *node.Node) error {
```

Comment:
Every error return in this function needs to return the actual `err` as well as the additional error message, example:
```
return fmt.Errorf("Failed due to some message, Err: %v", err)
```

Reply:
Done.